src/pages/dashboard.js

```diff
@@ -1,153 +0,0 @@
-import React, { useEffect, useState } from 'react';
-import {Alert, Box, Container, Grid} from '@mui/material';
-
-import ViewListIcon from "@mui/icons-material/ViewList";
-import BitcoinIcon from '@mui/icons-material/CurrencyBitcoin';
-import LocationSearchingIcon from '@mui/icons-material/LocationSearching';
-import PaymailIcon from '@mui/icons-material/Message';
-
-import { AdminChart } from '../components/dashboard/chart';
-import { DashboardLayout } from '../components/dashboard-layout';
-import { useUser } from "../hooks/user";
-import AdminLogin from "./admin-login";
-import { AdminCard } from "../components/dashboard/card";
-import { UtxosByType } from "../components/dashboard/utxos-by-type";
```

Comment:
I believe that those components should also be removed

Reply:
Removed